cmd/podman/login.go

```diff
@@ -75,7 +75,7 @@ func loginCmd(c *cli.Context) error {
 	}
 	sc.DockerInsecureSkipTLSVerify = !c.BoolT("tls-verify")
 	if c.String("cert-dir") != "" {
-		sc.DockerCertPath = filepath.Join(c.String("cert-dir"), server)
+		sc.DockerCertPath = filepath.Join(c.String("cert-dir"))
```

Comment:
do we need the join still?  Or can we just use the straight string?

Reply:
Woops, got excited and totally missed that. Fixed.